pgrx-macros/src/lib.rs

```diff
@@ -1177,12 +1173,7 @@ pub fn pg_aggregate(_attr: TokenStream, item: TokenStream) -> TokenStream {
     }
 
     let parsed_base = parse_macro_input!(item as syn::ItemImpl);
-    wrapped(parsed_base).unwrap_or_else(|e| {
-        let msg = e.to_string();
-        TokenStream::from(quote! {
-          compile_error!(#msg);
-        })
-    })
+    wrapped(parsed_base).unwrap_or_else(|e| e.to_compile_error().into())
```

Comment:
This diff is applied inconsistently: I could be hitting all the call sites that originally were
```rust
 wrapped(parsed_base).unwrap_or_else(|e| {
        let msg = e.to_string();
        TokenStream::from(quote! {
          compile_error!(#msg);
        })
    })
```